packages/webapp/src/inductions/components/induction-journey-container.tsx

```diff
@@ -0,0 +1,64 @@
+import React from "react";
```

Comment:
Prob not quite evident from this PR, but I'm planning to reuse this component in a few places in the induction process.